vspace/src/main/java/edu/asu/diging/vspace/core/services/impl/FileApiManager.java

```diff
@@ -0,0 +1,109 @@
+package edu.asu.diging.vspace.core.services.impl;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+
+import org.apache.tika.Tika;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Service;
+
+
+import edu.asu.diging.vspace.core.data.FileRepository;
+import edu.asu.diging.vspace.core.exception.FileStorageException;
+import edu.asu.diging.vspace.core.factory.IFileFactory;
+import edu.asu.diging.vspace.core.file.IStorageEngine;
+import edu.asu.diging.vspace.core.model.IVSFile;
+import edu.asu.diging.vspace.core.model.impl.VSFile;
+import edu.asu.diging.vspace.core.services.IFileApiManager;
+
+@Service
+public class FileApiManager implements IFileApiManager {
```

Comment:
Why is this called "File**Api**Manager"? What does the API refer to? Unless there is a good reason, it should just be `FileManager`.